packages/contracts/contracts/plugin/PluginSetupProcessor.sol

```diff
@@ -364,7 +363,7 @@ contract PluginSetupProcessor is DaoAuthorizable {
                 _dao,
                 _updateParams.plugin,
                 _currentHelpers,
-                oldVersion,
```

Comment:
Don't we need the release? Or we assume that the plugin setup will always be on the same release, I guess